### PR TITLE
Kotlin 1.2.30 updates

### PR DESCRIPTION
**Pull request by jaredsburrows**
Previous: https://github.com/android/android-ktx/pull/380

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,7 +79,7 @@ dependencies {
     metalava 'com.android.tools.lint:lint-checks:26.1.0-beta1'
     metalava 'com.android.tools.lint:lint-gradle:26.1.0-beta1'
     metalava 'com.android.tools.lint:lint:26.1.0-beta1'
-    metalava 'org.jetbrains.kotlin:kotlin-stdlib:1.2.21'
+    metalava "org.jetbrains.kotlin:kotlin-stdlib:$versions.kotlin"
```
Comment (jaredsburrows) [inline build.gradle]: This was missing the `$versions.kotlin` constant.

File: build.gradle [hunk 2]
```diff
@@ -14,7 +14,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.0-beta3'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$versions.kotlin"
```
Comment (jaredsburrows) [inline build.gradle]: `{}` are not needed.

File: build.gradle (hunk 1 above)
Comment (JakeWharton) [reply build.gradle]: This version is specifically pinned to the version metalava uses.

**Comment by JakeWharton** [comment]
I prefer `{` and `}` for expression templates (like Kotlin requires) because its far easier to read.

**Pull request closed without merging**